lists2safebrowsing.py

```diff
@@ -9,18 +9,18 @@
 import tempfile
 import time
 import urllib2
-import urlparse
 
 import boto.s3.connection
 import boto.s3.key
 
 # bring a URL to canonical form as described at
-# https://developers.google.com/safe-browsing/developers_guide_v2
+# https://developers.google.com/safe-browsing/v4/urls-hashing#canonicalization
```

Comment:
I would suggest this URL instead: https://web.archive.org/web/20160422212049/https://developers.google.com/safe-browsing/developers_guide_v2#Canonicalization

because we're not following the V4 spec, we're following the V2.2 spec (now unfortunately removed from the Google site) in shavar.